s3watcher/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+get_function_name() {
+    echo $(aws cloudformation list-stack-resources --stack-name $1 \
```

Comment:
Let's remember to always pass in the `--profile` explicitly with every `aws` command?